session/session.go

```diff
@@ -1088,6 +1091,99 @@ func (s *session) Execute(ctx context.Context, sql string) (recordSets []sqlexec
 	return
 }
 
+// Parse parses a query string to raw ast.StmtNode.
+func (s *session) Parse(ctx context.Context, sql string) ([]ast.StmtNode, error) {
```

Comment:
Add tracing for this method?

Reply:
Should `execute` be implemented by `Parse` and `ExecuteStmt`?

Reply:
s.ParseSQL has tracing already.